README.md

```diff
@@ -76,6 +76,31 @@ jobs:
 
 Refer to the [Azure PowerShell](https://github.com/azure/powershell) Github action to run your Azure PowerShell scripts.
```

Comment:
Before any of these changes are merged, we need a mention about the new "environment" parameter being introduced into the action.yml which supports all the new sovereign clouds and AzureStack.

Reply:
Added an introduction for environments parameter in the description for the Action. I am not too sure which sovereign clouds or what other capabilities you are supporting, so please updated based on context or let me know what to change